src/elements/content-sidebar/versions/VersionsSidebarContainer.js

```diff
@@ -163,9 +166,11 @@ class VersionsSidebarContainer extends React.Component<Props, State> {
         this.mergeResponse(data);
     };
 
-    handleFetchError = (): void => {
+    handleFetchError = (error: Object): void => {
+        const showVersionErrorWithUpsell = this.props.showVersionErrorWithUpsell && error.status === 403;
```

Comment:
Can you de-structure the prop first (it's optional, right?). Untested code approach: 
```suggestion
        const { showVersionErrorWithUpsell }  = this.props;
        const shouldShowVersionErrorWithUpsell = !!showVersionErrorWithUpsell && error.status === 403;
```